.gitignore

```diff
@@ -6,7 +6,6 @@ dist
 docs/_build/
 fastavro.egg-info
 fastavro/*.so
-fastavro/_*.c
```

Comment:
When I do a `make` or `make fresh` I still get `_*.c` files. We should keep this gitignore in or remove those files from being generated if they aren't supposed to be.

Reply:
I'll undo this change. It's left over from when I thought I should check in .c files.